gen/cycle.ml

```diff
@@ -839,7 +839,7 @@ let set_same_loc st n0 =
                       CoSet.set_cell has a case for pairs.
                       However increment of current value is by 2 *)
                    let cell = CoSt.get_cell st in
-                   assert (Array.length cell=2) ;
+                   assert (Array.length cell>=2) ;
```

Comment:
why >= 2? should pairs not always have cells of length 2?

Reply:
Correct, yet with Neon cells usually larger than that. IIUC, for pairs cell length must be at least 2, so when paired with Neon which has cell length of 4 and even more we still satisfy requirement for pairs, but current assert condition limit us with upper boundary.